Composer/ScriptHandler.php

```diff
@@ -186,30 +180,11 @@ public static function installRequirementsFile(CommandEvent $event)
         $appDir = $options['symfony-app-dir'];
         $fs = new Filesystem();
 
-        $newDirectoryStructure = static::useNewDirectoryStructure($options);
-
-        if (!$newDirectoryStructure) {
-            if (!static::hasDirectory($event, 'symfony-app-dir', $appDir, 'install the requirements files')) {
-                return;
-            }
-            $fs->copy(__DIR__.'/../Resources/skeleton/app/SymfonyRequirements.php', $appDir.'/SymfonyRequirements.php', true);
-            $fs->copy(__DIR__.'/../Resources/skeleton/app/check.php', $appDir.'/check.php', true);
-        } else {
-            $binDir = $options['symfony-bin-dir'];
-            $varDir = $options['symfony-var-dir'];
-            if (!static::hasDirectory($event, 'symfony-var-dir', $varDir, 'install the requirements files')) {
-                return;
-            }
-            if (!static::hasDirectory($event, 'symfony-bin-dir', $binDir, 'install the requirements files')) {
-                return;
-            }
-            $fs->copy(__DIR__.'/../Resources/skeleton/app/SymfonyRequirements.php', $varDir.'/SymfonyRequirements.php', true);
-            $fs->copy(__DIR__.'/../Resources/skeleton/app/check.php', $binDir.'/symfony_requirements', true);
-            $fs->remove(array($appDir.'/check.php', $appDir.'/SymfonyRequirements.php', true));
-
-            $fs->dumpFile($binDir.'/symfony_requirements', '#!/usr/bin/env php'.PHP_EOL.str_replace(".'/SymfonyRequirements.php'", ".'/".$fs->makePathRelative($varDir, $binDir)."SymfonyRequirements.php'", file_get_contents($binDir.'/symfony_requirements')));
-            $fs->chmod($binDir.'/symfony_requirements', 0755);
+        if (!static::hasDirectory($event, 'symfony-app-dir', $appDir, 'install the requirements files')) {
+            return;
         }
+        $fs->copy(__DIR__.'/../Resources/skeleton/app/SymfonyRequirements.php', $appDir.'/SymfonyRequirements.php', true);
+        $fs->copy(__DIR__.'/../Resources/skeleton/app/check.php', $appDir.'/check.php', true);
```

Comment:
should they be installed in the ``var`` folder in 3.0 ? this is what the code was doing previously.